website/templates/index.html

```diff
@@ -26,7 +26,7 @@ <h1 class="page-header">{% trans "Find Bugs, Win Points and Prizes" %}</h1>
     <div class="container">
         <center>
             <div class="row">
-                <div class="col-sm-4">    
+                 <div class="col-sm-3 cborder">    
```

Comment:
Just because we need the style somewhere else, that doesn't mean we have to add a new class for it. Almost every element in DOM can be targetted by the classes that are already there. Adding more classes is of no use.